frontend/src/v5/ui/v4Adapter/overrides/preview/previewDetails.overrides.ts

```diff
@@ -110,6 +115,12 @@ export default css`
 					margin: 0;
 					box-sizing: border-box;
 					height: 100%;
+					width: 100%;
```

Comment:
Not saying we do this in this PR as it should probably be a new PR, but might it be a good idea to add the CSS for truncating elements to the theme, a bit like how we do typography? Since we seem to duplicate these lines of CSS a lot. Of course there is the V4 Truncate component, but I don't think this can be used everywhere.

Reply:
Go ahead and list all the strings that overflow their parents, and then we may create a new issue to address them all or, as they all target just the styling (so the eventual PR will not be too hard to review), I might as well fix them all here. Would that be ok @sanmont3drepo?

Reply:
Created an issue for this #3827